project-fob/Views/Home/MeetingPageHost.cshtml

```diff
@@ -1,192 +1,79 @@
-		var counter = 0;
-		function clickHandler(e) {
-
+﻿<script type="text/javascript">
+    
+    var counter = 0;
+    function reset() {
 			$.ajax({ //Needs to check if the database is running/ can connect. If not should dc from room or tell the user
 				type: "POST",
-				url: '@Url.Action("Reset", "Host")',
-				success: function (result) {
-					//alert("successfull reset");
-					var split = result.split(',')
-					updateChart(split[0], split[1]);
-				},
-				error: function () {
-					alert("Connection error");
-				}
-			});
-
-			button = new google.visualization.PieChart(document.getElementById('button'));
-			button.draw(dataButton, clickedButton);
-			google.visualization.events.addListener(button, 'select', clickHandler);
-			setTimeout(function () {
-				button = new google.visualization.PieChart(document.getElementById('button'));
-				button.draw(dataButton, optionsButton);
-				google.visualization.events.addListener(button, 'select', clickHandler);
-			}, 50);
-            
-		}
-
-		function buttonBackToNormal() {
-			var button = new google.visualization.PieChart(document.getElementById('button'));
-			button.draw(dataButton, optionsButton);
-		}
-
-		if (!isItDrawing) {
-			runUpdate();
-			isItDrawing = true;
-		}
-		var errorCounter = 0;
-		function runUpdate() { // host controller, refresh
-			setTimeout(runUpdate, 1000);
-			$.ajax({
-				type: "POST",
-				url: '@Url.Action("Refresh", "Host")',
+                url: '@Url.Action("Reset", "Host")',
+                data: { meetingIdString : "@ViewBag.meetingid" },
                 success: function (result) {
 
-                    updateChart(result, result);
-                    //alert("after if");
-                    var name = "Click for Next Topic, Fob count: " + result;
-						errorCounter = 0;
-						dataButton = google.visualization.arrayToDataTable([
-						['Language', 'Speakers (in millions)'],
-						[name, 100],
-						]);
-						button = new google.visualization.PieChart(document.getElementById('button'));
-						button.draw(dataButton, optionsButton);
-						google.visualization.events.addListener(button, 'select', clickHandler);
 				},
 				error: function () {
-					errorCounter++;
-					if (errorCounter >= 30) {
-						alert("Connection error. Returning to home page")
-						errorCounter = 0;
-						window.location = "index";
-					}
+					alert("Connection error");
 				}
 			});
 		}
+    
+        	var errorCounter = 0;
+    function runUpdate() { // host controller, refresh
+			        setTimeout(runUpdate, 1000);
+			        $.ajax({
+				        type: "POST",
+                        url: '@Url.Action("Refresh", "Host")',
+                        data: { meetingIdString : "@ViewBag.meetingid" },
+                        success: function (result) {
+                            //alert("hey");
+                            //This still need fixed
+                            //updateChart(result, result);
```

Comment:
Commented out code like this is a bit icky

Reply:
I told this guy to leave multiple times but it doesn't budge.